src/main/java/org/assertj/core/util/introspection/Introspection.java

```diff
@@ -77,14 +77,15 @@ private static Method findGetter(String propertyName, Object target) {
     String capitalized = propertyName.substring(0, 1).toUpperCase(ENGLISH) + propertyName.substring(1);
     // try to find getProperty
     Method getter = findMethod("get" + capitalized, target);
-    if (getter != null) return getter;
+    if (getter != null && !Modifier.isStatic(getter.getModifiers())) return getter;
```

Comment:
let' s extract a static `isValidGetter(Method method)` method to capture the `getter != null && !Modifier.isStatic(getter.getModifiers())` repeated 3 times, will be more readable too.

Reply:
Thanks for the suggestions. I've pushed a commit to do this.